src/Compilers/CSharp/Test/Syntax/Parsing/FileModifierParsingTests.cs

```diff
@@ -2500,33 +2551,17 @@ public void TopLevelVariable_02()
     [Fact]
```

Comment:
>Fact [](http://example.com/codeflow?start=5&length=4)

Consider testing with:
```
[Theory]
[InlineData(LanguageVersion.CSharp10)]
[InlineData(LanguageVersionFacts.CSharpNext)]
```

Reply:
Fixed this and the similar previous comment.